patroni/dcs/__init__.py

```diff
@@ -1092,9 +1094,9 @@ def _get_members_slots(self, name: str, role: str) -> Dict[str, Dict[str, str]]:
             * Conflicting slot names between members are found
 
         :param name: name of this node.
-        :param role: role of this node, if this is a ``primary`` or ``standby_leader`` return list of members
-                     replicating from this node. If not then return a list of members replicating as cascaded
-                     replicas from this node.
+        :param role: role of this node, ``primary``, ``standby_leader``, or ``replica``.
+        :param nofailover: ``True`` if this node is tagged to not be a failover candidate, ``False`` otherwise.
+        :param can_advance_slots: ``True`` if PostgreSQL major version is v11+, ``False`` otherwise.
```

Comment:
i would better use the same docstring as in `Cluster._merge_permanent_slots()`

```
``True`` if ``pg_replication_slot_advance()`` function is available,
                                  ``False`` otherwise.
 ```